bundles/org.openhab.binding.myuplink/README.md

```diff
@@ -0,0 +1,79 @@
+
+## Supported Things
+
+This binding provides two thing types:
+
+| Thing/Bridge        | Thing Type          | Description                                                                                   |
+|---------------------|---------------------|-----------------------------------------------------------------------------------------------|
+| bridge              | account             | cloud connection to a myUplink user account                                                   |
+| thing               | genericDevice       | the physical heatpump which is connected to myUplink                                          |
+
+
+## Discovery
+
+Auto-discovery is supported and will discover all heatpumps within an account and also detect the specific channels supported by the model.
+
+## Bridge Configuration
+
+The following configuration parameters are available for the binding/bridge:
+
+| Configuration Parameter | Required | Description                                                                                                                                                                                 |
+|-------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| clientId                | yes      | The clientId to login at myUplink cloud service. This is some kind of UUID. Visit https://dev.myuplink.com/apps to generate login credentials.                                              |
+| clientSecret            | yes      | The secret which belongs to the clientId.                                                                                                                                                   |
+| dataPollingInterval     | no       | Interval (seconds) in which live data values are retrieved from the Easee Cloud API. (default = 60)                                                                                         |
+
+## Thing Configuration
+
+It is recommended to use auto discovery which does not require further configuration.
+If manual configuration is preferred you need to specify configuration as below.
+
+| Configuration Parameter | Required | Description                                                                                                            |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------|
+| deviceId                | yes      | The id of the heatpump that will be represented by this thing. Can be retrieved via API call or autodiscovery.         |
+
+
+## Channels
+
+The binding only supports all channels which are explicitely exposed by the myUplink API.
+Depending on your model and additional hardware the channels might be different. 
+Thus no list is provided here.

```

Comment:
Have to look into the details, but in general it is best to show the channels. The above warning is already good enough for users to know not all channels would be available.

Reply:
I cannot provide any channels because the binding is generic and there are more than 50 different heat pumps (from different vendors) supported. They do not have a common set of channels. I would need to own one of each to add all the channels. In total this would be a list containing mor than 10.000 channels. I do not think that this would be useful for anybody.

Reply:
No base set of common channels? I would have thought each heatpump would have a setpoint temperature for instance. Maybe not all, but just a few as example would already give some insights into the convention, structure that gives some guidance. If this also doesn’t make sense we will have to leave it as is.

Reply:
Unfortunately this is not the case. This is a multi-vendor API and although some channels like setpoint temperature exist for all models it will have different implementations e.g.
- different channel ID
- different min/max values
- different step values

Well this is the drawback of a generic API.

Reply:
@jlaur or @lolodomo are you aware of any binding with a similar generic approach and how this is handled regading documentation?